src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs

```diff
@@ -608,9 +626,9 @@ private string PreProcessAuthScope()
             return mappedScope;
         }
 
-        private bool IsUsingInteractiveAuthentication()
+        private bool IsBrowserPopUpInteractiveAuthentication()
```

Comment:
Why renaming this method? My concern is WAM is also an interactive authentication flow but it's not browser based

Reply:
rename it to distinguish with
```
        private bool IsInteractiveAuthenticationFlow()
        {
            return ParameterSetName.Equals(UserParameterSet);
        }
```

this method is used to check if need to write warning when popup is needed.
                if (!commonUtilities.IsDesktopSession() && IsPopUpInteractiveAuthenticationFlow())
                {
                    WriteWarning(Resources.InteractiveAuthNotSupported);
                    return;
                }